seastar/src/lib.rs

```diff
@@ -16,3 +16,23 @@ pub(crate) use seastar_test_guard::acquire_guard_for_seastar_test;
 
 pub use config_and_start_seastar::*;
 pub use preempt::*;
+
+/// A macro that runs the `main` function's contents
+/// in Seastar's runtime. **Only** `main` is allowed
+/// to use this macro - Seastar apps may only use
+/// one `AppTemplate` instance at a time.
```

Comment:
The docstring should start with a block that shortly describes given item, then an empty comment line should follow, and then the rest of the docstring. `cargo doc` uses the first block as a summary of given item. This summary is e.g. displayed on the doc page of the encompassing module, on the list of module's items.

So, like this:

```rust
/// A macro that runs the `main` function's contents
/// in Seastar's runtime.
/// **Only** `main` is allowed to use this macro -
/// Seastar apps may only use one `AppTemplate`
/// instance at a time.
```

Reply:
Ah, I see.